UnityProject/Assets/Scripts/UI/Right click/RadialMenuSpawner.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RadialMenuSpawner : MonoBehaviour {
+
+	public static RadialMenuSpawner ins;
+	public RadialMenu menuPrefab;
+
+	void Awake(){
```

Comment:
See below about the need to check if the instance has already been assigned.

i.e. static instances need to be set up like this:
```cs
void Awake(){
    if(ins == null){
        ins = this;
    } else {
       Destroy(this);
    }
}
```